css/style.css

```diff
@@ -27,14 +27,23 @@ body{
 .play{background: url('../img/play.png') ;}
 .pause{background: url('../img/pause.png') ;}
 
+#track-name{
```

Comment:
Need to add little padding so the characters are not sticking to the end in case of large track names.